osu.Game/Rulesets/Scoring/DrainingHealthProcessor.cs

```diff
@@ -47,6 +47,8 @@ public class DrainingHealthProcessor : HealthProcessor
         private double targetMinimumHealth;
         private double drainRate = 1;
 
+        private int breakendcounter;
```

Comment:
```suggestion
        private bool judgedAfterBreak;
```
is probably what you want

Reply:
Fixed ( naming is different ) in 61be2f5.

Reply:
`breakend` doesn't make much sense when talking about draining health after first hit object being hit (we're at the beginning, not after a break end).

Also please follow our naming style for consistency and better code readability (each word from second onwards starts with capital letter): 
`Judged after break` field for example becomes `judgedAfterBreak` not `judgedafterbreak`